pkgs/applications/networking/firewalk/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, fetchurl
+, libnet
+, libpcap
+, libdnet
+}:
+
+stdenv.mkDerivation rec {
+    pname = "firewalk";
+    version = "5.0";
+
+    src = fetchurl {
+        url = "https://salsa.debian.org/pkg-security-team/firewalk/-/archive/debian/5.0-5/firewalk-debian-5.0-5.tar.gz";
+        hash = "sha256-luFTi4DnXbiNDI2uPgbtF4Y9m/SdxQBs+Dfdmck7mFU=";
+    };
+
+    buildInputs = [ libnet libpcap libdnet ];
+
+    meta = with lib; {
+        description = "Gateway ACL scanner";
+        homepage = "http://www.packetfactory.net/firewalk";
```

Comment:
This page is dead.
```suggestion
        homepage = "http://packetfactory.openwall.net/projects/firewalk/";
```

Reply:
changes accepted and made.

Reply:
does an entry still apply after the recent changes?

Reply:
Could you please read this comment and document? Thanks.

Reply:
@natsukium, Thanks for the recommendation. Have i done the pkgs/by-name correctly?